roles/variables/vars/main.yml

```diff
@@ -1573,8 +1577,17 @@ control_center_properties:
       confluent.controlcenter.rest.authentication.method: BEARER
       public.key.path: "{{rbac_enabled_public_pem_path}}"
       confluent.metadata.bootstrap.server.urls: "{{mds_bootstrap_server_urls}}"
+  rbac_ldap:
+    enabled: "{{rbac_enabled and not oauth_enabled}}"
+    properties:
       confluent.metadata.basic.auth.user.info: "{{ control_center_ldap_user | default('c3') }}:{{ control_center_ldap_password | default('pass') }}"
       confluent.metadata.http.auth.credentials.provider: BASIC
+  rbac_oauth:
+    enabled: "{{rbac_enabled and oauth_enabled}}"
+    properties:
+      confluent.metadata.oauthbearer.token.endpoint.url: "{{ oauth_token_uri }}"
+      confluent.metadata.oauthbearer.login.client.id: "{{ control_center_oauth_user }}"
+      confluent.metadata.oauthbearer.login.client.secret: "{{ control_center_oauth_password }}"
```

Comment:
why is confluent.metadata.http.auth.credentials.provider: OAUTHBEARER not needed here

Reply:
reference https://confluentinc.atlassian.net/wiki/spaces/CPFundamentals/pages/3232632615/Confluent+Platform+Configs+for+OAuth?focusedCommentId=3478457507